spec/string_builder_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "build_string"
+
+describe BuildString do
+  subject(:jack_house) {described_class.new }
+  
+  describe BuildString do
+      # Reminder: use . in front of class methods, and # in front of instance methods.
+      describe ".rhyme" do 
```

Comment:
Based on your recent update rhyme is not a class method anymore. Following your comment in line 7 , line 8 should be `#rhyme`.